content/blog/2025-01-13-benny-vasquez-qa.md

```diff
@@ -0,0 +1,70 @@
+
+{{< youtube id="SFHTc4V2n_I" width="65%" height="36%" autoplay="false" controls="true" mute="false" title="Meet the board: benny Vasquez" >}}
+
+## Learn more
+
+The AlmaLinux OS Foundation's board is community elected, by members of the AlmaLinux OS Foundation. If you use AlmaLinux, contribute to AlmaLinux, or support AlmaLinux in some way, we encourage you to [join the foundation](/members/). Individual membership is free, because you shouldn't have to pay to have a voice in the future of something you care about. 
+
+See the other interviews here:
+
+* In September, we interviewed [Jack Aboutboul](https://www.linkedin.com/in/jackaboutboul/): [Meet the board - Jack Aboutboul](/blog/2024-09-12-meet-the-board-jack/)
+* In September, we interviewed [Jun Yoshida](https://www.linkedin.com/in/jun-yoshida-6b4b5a16/): [Meet the board - Jun Yoshida](blog/2024-09-23-meet-the-board-jun/)
+* In October, we interviewed [Alex Iribarren](https://www.linkedin.com/in/iribarren/): [Meet the Board: Alex Iribarren](/blog/2024-10-08-meet-the-board-alex/)
+
+We also had a panel of board members at AlmaLinux Day: Germany in 2024 where board members talked about everything from how they got involved in AlmaLinux to their feelings about the future of open source, which you can watch on YouTube [Meet the board!](https://www.youtube.com/watch?v=PP3OPvmUwTs)
+
+If you'd like to learn more about our board and elections, you can check out [our Wiki](https://wiki.almalinux.org/election2023.html)! You can also watch the other videos in this series in [this YouTube playlist](https://www.youtube.com/playlist?list=PLqKzWfm6zyluP2R_6MTPvYJt20bwitQZJ).
+
+## Transcript
+
+_Note: This transcript has been edited for clarity and readability._
+
+Noam Alum: Hey everyone. I'm Noam Alum, a member of the AlmaLinux community. I'm here with benny from the AlmaLinux Foundation. And we are going to introduce you a bit more to the AlmaLinux community. How are you doing today, benny?
+
+benny Vasquez: I'm good. Thanks, Noam. I am very excited to be here.
+
+Noam Alum: Great. Let's jump right into it. What got you into open source in general and Linux specifically?
+
+benny Vasquez: So my first Linux distribution was CentOS. I guess technically, it was Ubuntu when I checked my email and my boyfriend's computer once. But when I got into web hosting, the primary OS that we used was CentOS, so that was my first. And it has had a very special place in my heart ever since then. Later on, when I started to learn about open source, I loved the community nature of open source in general, I loved the idea that just a bunch of people that care about something could work together on it and, like, allow your passion to drive something cool. And eventually that led me to AlmaLinux. 
```

Comment:
```suggestion
benny Vasquez: So my first Linux distribution was CentOS. I guess technically, it was Ubuntu when I checked my email on my boyfriend's computer once. But when I got into web hosting, the primary OS that we used was CentOS, so that was my first. And it has had a very special place in my heart ever since then. Later on, when I started to learn about open source, I loved the community nature of open source in general, I loved the idea that just a bunch of people that care about something could work together on it and, like, allow your passion to drive something cool. And eventually that led me to AlmaLinux. 
```